x-pack/plugins/security/server/audit/audit_events.ts

```diff
@@ -218,6 +220,7 @@ const savedObjectAuditTypes: Record<SavedObjectAction, EventType> = {
   saved_object_find: EventType.ACCESS,
   saved_object_add_to_spaces: EventType.CHANGE,
   saved_object_delete_from_spaces: EventType.CHANGE,
+  saved_object_open_point_in_time: EventType.CREATION,
```

Comment:
@legrego not sure which event type opening a PIT should be associated to?

Reply:
++ @legrego Would appreciate any feedback here as this is the first time I've touched audit logging (or the SO client wrappers from security/spaces). One thing in particular I was wondering was whether it would be an issue that `openPointInTimeForType` in the repository didn't accept a `namespace`.

I didn't think it needed one since we aren't really doing anything space-specific here, just proxying a PIT request to a SO type's backing indices. But want to make sure I'm not overlooking something important.

Reply:
> ++ @legrego Would appreciate any feedback here as this is the first time I've touched audit logging (or the SO client wrappers from security/spaces). 

Thanks for being willing to take these changes on! This can be a twisted web of complexity, so I appreciate you giving this a shot. We're more than willing to help out along the way 💚 

> One thing in particular I was wondering was whether it would be an issue that `openPointInTimeForType` in the repository didn't accept a `namespace`.
> 
> I didn't think it needed one since we aren't really doing anything space-specific here, just proxying a PIT request to a SO type's backing indices. But want to make sure I'm not overlooking something important.

Technically speaking, I don't think the repository needs the `namespace`, but the wrappers share a common interface with the repository, and the wrappers need this information in order correctly authorize and audit access to this functionality. For the sake of completeness and consistency, I think we should pass it all the way down, even if we aren't using it at this time.

Reply:
> For the sake of completeness and consistency, I think we should pass it all the way down, even if we aren't using it at this time.

Okay that works for me if it makes things easier - I can add it as a parameter and just ignore it in the repository